src/datascience-ui/data-explorer/cellFormatter.css

```diff
@@ -1,9 +1,13 @@
 .number-formatter {
     text-align: right;
+    text-overflow: ellipsis;
+    overflow: hidden;
 }
 
 .cell-formatter {
     /* Note: This is impacted by the RowHeightAdjustment in reactSlickGrid.tsx */
     margin: 0px 0px 0px 0px;
     text-align: right;
+    text-overflow: ellipsis;

```

Comment:
Yay, done properly with CSS....

Reply:
Yeah, I was really happy to discover today that this was supported in CSS